pages.it/linux/apt-get.md

```diff
@@ -0,0 +1,36 @@
+# apt-get
+
+> Servizio di gestione dei pacchetti per Debian e Ubuntu
+> Cerca i pacchetti usando `apt-cache`.
+
+- Aggiorna la lista dei pacchetti e delle loro versioni disponibili (prima di altri comandi `apt-get` è consigliato eseguire questo comando):
```

Comment:
(minor, for readability) Consider using "è consigliato eseguire questo comando prima di altri comandi `apt-get`"

Reply:
the same recommendation applies also to "apt"